### DISCUSSION TITLE
Ansible lint everything passes

### DISCUSSION THREAD

**Discussion by failattu** [ansible/ansible-lint#1492] [Q&A]
##### Summary
We are running the latest Ansible lint from pip and its passing everything, no complains about anything when we ansible-lint 

##### Issue Type

- Bug Report

##### Ansible and Ansible Lint details
```console (paste below)
ansible --version
ansible 2.9.12
ansible-lint --version
ansible-lint 5.0.5 using ansible 2.9.12
```

- ansible installation method: one of source, pip, OS package
pip
- ansible-lint installation method: one of source, pip, OS package
pip

##### OS / ENVIRONMENT
Ubuntu

##### STEPS TO REPRODUCE

pip3 install --user ansible-lint
ansible-lint 

+ ansible-lint /home/jenkins/workspace/xy_
[Pipeline] }

```

##### Desired Behaviour
We get alarms

##### Actual Behaviour
Nothing complains about anything

**Comment by ssbarnea** [comment]
That was not a proper bug report so I converted it to a discussion. Please do not use the issue tracker for support.

**Reply by failattu** [reply]
Hi,
I don't know what else to say about it. We run it to many ansible roles, and all of them end up passing even though they should complain from many things.

**Reply by failattu** [reply]
and we are freshly installing ansible-lint from pip, we have had it working for 2 years with same pipeline giving us the error statements.
previously:  
Creating new machine -> deploy lint -> pull git -> run lint -> lint gives errors we fix
now:
Creating new machine -> deploy lint -> pull git -> run lint -> lint passes
